ext/MitsubaLoader/CMitsubaMaterialCompilerFrontend.h

```diff
@@ -0,0 +1,34 @@
+#ifndef __C_MITSUBA_MATERIAL_COMPILER_FRONTEND_H_INCLUDED__
+#define __C_MITSUBA_MATERIAL_COMPILER_FRONTEND_H_INCLUDED__
+
+#include "../../ext/MitsubaLoader/CElementBSDF.h"
+#include <irr/asset/material_compiler/IR.h>
+
+#define DERIV_MAP_FLOAT32
+
+namespace irr
+{
+namespace ext
+{
+namespace MitsubaLoader
+{
+    struct SContext;
+
+class CMitsubaMaterialCompilerFrontend
+{
+    const SContext* m_loaderContext;
+
+    using tex_ass_type = std::tuple<core::smart_refctd_ptr<asset::ICPUImageView>, core::smart_refctd_ptr<asset::ICPUSampler>, float>;
+    core::unordered_map<core::smart_refctd_ptr<asset::ICPUImage>, core::smart_refctd_ptr<asset::ICPUImageView>> m_treeCache;
```

Comment:
what's with the name of this variable?

Reply:
not sure why its even there, it's not being used. No idea really..